src/components/referrals/ReferralsComponent.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+    <div
+        class="referralsBox"
+        @submit.prevent="addTest">
+        <h3 class="referralsBox__header">Dodaj skierowanie:</h3>
+        <form class="referralsBox__form referralsForm">
+            <div class="inputBox">
+                <label for="doctorData"> Lekarz:</label>
```

Comment:
You've got in ImportReferralsComponent.vue element with id = doctorData and also here. Its wrong and will be problem with styles and accessbility. You sould change it.

Reply:
i changed id.